modules/test/playwright/tests/object-web/utils/mockObjectFieldsObjectEntry.ts

```diff
@@ -0,0 +1,362 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {getRandomInt} from '../../../utils/getRandomInt';
+import getRandomString from '../../../utils/getRandomString';
+
+type objectFieldBusinessType = (typeof objectFieldBusinessTypes)[number];
+
+type objectFieldBusinessTypesLabelName = {
+	[K in objectFieldBusinessType]: objectFieldLabelName;
+};
+
+type objectFieldLabelName = {
+	label: string;
+	name: string;
+};
+
+const objectFieldBusinessTypes = [
+	'autoIncrement',
+	'boolean',
+	'decimal',
+	'encrypted',
+	'integer',
+	'longInteger',
+	'longText',
+	'multiselectPicklist',
+	'picklist',
+	'precisionDecimal',
+	'richText',
+	'text',
+] as const;
+
+function mockObjectEntry(
+	objectFieldBusinessTypesInfo: objectFieldBusinessTypesLabelName,
+	listTypeEntries: string[]
+) {
+	return {
+		[objectFieldBusinessTypesInfo.boolean.name]: Math.random() < 0.5,
+		[objectFieldBusinessTypesInfo.decimal.name]: parseFloat(
+			Math.random().toFixed(10)
+		).toString(),
+		[objectFieldBusinessTypesInfo.encrypted.name]: getRandomString(),
+		[objectFieldBusinessTypesInfo.integer.name]: Math.floor(
+			Math.random() * 100
+		),
+		[objectFieldBusinessTypesInfo.longInteger.name]: getRandomInt(),
+		[objectFieldBusinessTypesInfo.longText.name]: getRandomString(),
+		[objectFieldBusinessTypesInfo.multiselectPicklist.name]: [
+			listTypeEntries[0],
+			listTypeEntries[1],
+		],
+		[objectFieldBusinessTypesInfo.picklist.name]: {
+			key: listTypeEntries[0],
+		},
+		[objectFieldBusinessTypesInfo.precisionDecimal.name]: parseFloat(
+			Math.random().toFixed(15)
+		).toString(),
+		[objectFieldBusinessTypesInfo.richText.name]: getRandomString(),
+		[objectFieldBusinessTypesInfo.text.name]: getRandomString(),
+	};
```

Comment:
Hey @igor-franca, to avoid repeating code, you could destructure this `objectFieldBusinessTypesInfo`?

```ts
	const {
		boolean,
		decimal,
		encrypted,
		integer,
		longInteger,
		longText,
		multiselectPicklist,
		picklist,
		precisionDecimal,
		richText,
		text,
	} = objectFieldBusinessTypesInfo;
```

Reply:
Also, I'm wondering if anyone else who would use this function and not pass at least 2 items in the listTypeEntries array, I think this function will break, so I recommend treating this possible error case like:

```ts
	if (listTypeEntries.length < 2) {
		throw new Error('listTypeEntries must contain at least two entries');
	}
```